### PR TITLE
Allow parameters in custom test functions

### PR DESCRIPTION
**Pull request by lordjabez**
This is to address issue #63.

### CODE REVIEW

File: tests/test_queries.py [hunk 1]
```diff
@@ -112,6 +112,19 @@ def test(value):
...
+def test_custom_with_params():
+    def test(value, minimum, maximum):
+        return minimum <= value <= maximum
+
+    query = where('val').test(test, 1, 10)
+
+    assert query({'val': 5})
+    assert not query({'val': 0})
+    assert not query({'val': 11})
+    assert not query({'val': '44'})
```
Comment (msiemens) [inline tests/test_queries.py]: You're using a string for `44`, not an int. This breaks the tests for Python3 and PyPy3 (see https://travis-ci.org/msiemens/tinydb/builds/68895961).

**Comment by msiemens** [comment]
There's a tiny mistake in a test, but otherwise it looks good!

**Comment by lordjabez** [comment]
Should I just remove that test? I only included the string comparison because there was a similar one for the previous test case.

**Comment by lordjabez** [comment]
Ahh, I see the issue now. I'll simply remove the test.

**Pull request merged**